src/ptf/testutils.py

```diff
@@ -1858,6 +1858,35 @@ def verify_packets_any(test, pkt, ports=[], device_number=0):
 
     test.assertTrue(received == True, "Did not receive expected pkt on any of ports %r for device %d" % (ports, device_number))
 
+def verify_packet_any_port(test, pkt, ports=[], device_number=0):
```

Comment:
I see. I don't mind adding this function, but then could you make sure the comment actually reflects what the function is doing? Right now I think it is just a copy-paste of `verify_any_packet_any_port`. It would be nice to say that packets received until the actual expected packet won't cause the test to fail.